src/pet.js

```diff
@@ -70,16 +70,21 @@ function Pet() {
     },
```

Comment:
In your checkUp() method, you can either do it as you did, with a couple of if statements that will return something, and so stop the code from running, or you can do a single if statement with a couple of else if in it. The second way would be more readable, but yours is also right.

Reply:
The else-if syntax made more sense to me and that's how I did it initially. However, my tests did not pass with it. Now that I've implemented it again, it works! Lol. Thank you for pointing it out.